config/runtime/tests/wifi-basic.jinja2

```diff
@@ -0,0 +1,25 @@
+{# This test verifies WiFi basic functionality:
+ # - Checks rfkill soft block/unblock
+ # - Performs a scan for nearby networks
+ # - Checks that monitor/managed modes can be enabled
+ # This test assumes there are APs within reach.
+-#}
+- test:
+    timeout:
+      minutes: 5
```

Comment:
Please allow overriding with `job_timeout` (see other test templates for reference)

Reply:
Right, thanks for the pointer. Done!